AppCenter/AppCenter/Internals/Model/MSAbstractLogPrivate.h

```diff
@@ -1,3 +1,18 @@
+#import <Foundation/Foundation.h>
+
+static NSString *const kMSBooleanTypedPropertyType = @"boolean";
```

Comment:
Can we group constants by usage? For example you can get property type constants together.